torchmetrics/functional/__init__.py

```diff
@@ -108,6 +109,7 @@
     "retrieval_reciprocal_rank",
     "roc",
     "rouge_score",
+    "sacrebleu_score",
```

Comment:
how about?
```suggestion
    "sacre_bleu_score",
```

Reply:
Yes, this looks reasonable! I change it accordingly :)